arith/gf2_128/src/gf2_ext128x8/neon.rs

```diff
@@ -75,7 +75,7 @@ impl Field for NeonGF2_128x8 {
 
     const SIZE: usize = 16 * 8;
 
-    const FIELD_SIZE: usize = 128 * 8; // in bits
+    const FIELD_SIZE: usize = 128; // in bits
```

Comment:
I think I caught a bug casually, this should be the GF2_ext128 size, not the size of GF2_ext128x8